src/toil/batchSystems/slurm.py

```diff
@@ -193,6 +193,15 @@ def prepareSbatch(self, cpu, mem, jobID):
         if cpu is not None:
             sbatch_line.append('--cpus-per-task={}'.format(int(math.ceil(cpu))))
 
+        # "Native extensions" for SLURM (see DRMAA or SAGA)
+        nativeConfig = os.getenv('TOIL_SLURM_ARGS')
+        if nativeConfig is not None:
+            logger.debug("Native SLURM options appended to sbatch from TOIL_SLURM_RESOURCES env. variable: {}".format(nativeConfig))
```

Comment:
% formatting isn't going to go away anytime soon. Letting the logger do the interpolation is  more concise and more efficient because the interpolation is performed only if the logging threshold is at debug.